mirrord/agent/src/runtime/crio.rs

```diff
@@ -47,7 +47,7 @@ impl CriOContainer {
                 Request::builder()
                     .method("GET")
                     .header("Host", "localhost")
-                    .uri(format!("http://localhost/{}", path))
+                    .uri(format!("http://localhost/{}", path.to_string().trim_start_matches('/')))
```

Comment:
The `api_get` is a private function so since you have changed the paths it can get I wouldn't do a second removal of `/`, and in this scenario `to_string` is completely unnecessary because the resulting `String` is being derefed back down to str for `trim_start_matches`.

Changing to `format!("http://localhost{path}")` and keeping the paths `/pause/{}` and `/unpause/{}` should also be fine because because `api_get` will probably remain private for it's existence

Reply:
Fixed